bin/tailf2kafka

```diff
@@ -87,10 +88,11 @@ end
 @files = {}
 @threads = {}
 @position_file = @settings[:tailf][:position_file]
-@flush_interval = @settings[:tailf][:flush_interval]
-@max_batch_lines = @settings[:tailf][:max_batch_lines]
+@flush_interval = @settings[:tailf].has_key?(:flush_interval) ? @settings[:tailf][:flush_interval] : 1
+@max_batch_lines = @settings[:tailf].has_key?(:max_batch_lines) ? @settings[:tailf][:max_batch_lines] : 1024
```

Comment:
don't you think 1024 is a lot of lines? Some files are getting low volumes, like postback events is about 100 lines per minute?